packages/fuselage/src/components/Options/useCursor.ts

```diff
@@ -67,7 +67,8 @@ export const useCursor = <
     value: unknown,
     label: unknown,
     selected?: unknown,
-    type?: OptionType[3]
+    type?: OptionType[3],
+    url?: unknown
```

Comment:
why url is `unknown` ?

Reply:
I was following the other fields and thought that since `label` was a string but it's typed as an unknown the `URL` should be equal.